### DISCUSSION TITLE
How to specify parent child streams in tap-rest-api-msdk ?

### DISCUSSION THREAD

**Discussion by silverbullet1** [meltano/sdk#2109] [Q&A] [answered]
Hi!

I have 2 streams X and Y.
My stream Y depends on X, i.e Y will iterate over all IDs that I got from X's stream. How do i specify this in the config ? Is this supported via SDK? or should it be handled in the tap

Thanks 😄

**Comment by edgarrmondragon** [comment, marked as answer]
Hi @silverbullet1!

The SDK [can do this](https://sdk.meltano.com/en/latest/parent_streams.html) in custom implementations, but I don't think `tap-rest-api-msdk` supports it. Worth raising an issue in their repo.

**Reply by edgarrmondragon** [reply]
I see you did!

https://github.com/Widen/tap-rest-api-msdk/issues/40
